frameworks/Rust/actix/src/main.rs

```diff
@@ -48,32 +47,26 @@ fn world_row(req: HttpRequest<State>) -> Box<Future<Item=HttpResponse, Error=Err
 fn queries(req: HttpRequest<State>) -> Box<Future<Item=HttpResponse, Error=Error>> {
     // get queries parameter
     let q = if let Some(q) = req.query().get("q") {
-        q.parse::<usize>().ok().unwrap_or(1)
+        q.parse::<u16>().ok().unwrap_or(1)
     } else {
         1
     };
```

Comment:
I've been playing around with rust a bit. These 5 lines could be:

`let q = req.query().get("q").unwrap_or("1").parse::<u16>().unwrap_or(1);`

Happens in updates too in this file, which is still using `<usize>`

Not a nit or anything, just enjoying messing around with the language.

Reply:
well, if this ```.unwrap_or("1")``` happen, then you would parse default string.

equivalent would be:
```rust
let q = req.query().get("q").unwrap_or(1).map(|v| v.parse::<u16>().unwrap_or(1));
```

Reply:
```rust
let q: u16 = req.query().get("q").map(|v| v.parse().unwrap_or(1)).unwrap_or(1);
```